templates/base/package.json

```diff
@@ -15,13 +15,21 @@
     "generate": "plop --plopfile generators/index.js",
     "analyze": "cross-env ANALYZE=true next build"
   },
+  "browserslist": [
+    "> 0.25%",
+    "not ie <= 11",
```

Comment:
Are we sure ?

Dont we want to support IE 11 ?

Reply:
This is <=, so IE11 is included